cfgov/jinja2/v1/_includes/organisms/table.html

```diff
@@ -49,7 +49,7 @@
 
 {% if value.has_data is not defined or value.has_data %}
 {% macro _render_cell(cell) %}
-    {{ (cell or '&nbsp;') | richtext | safe | trim | linebreaksbr }}
+    {{ (cell or '&nbsp;') | richtext | linebreaksbr | trim | safe }}
```

Comment:
For some reason the order of these filters was causing the string to no longer be safe when running in Python 3.6. This order seems to preserve desired behavior and desired rendering results.

Reply:
Not a blocker for this PR, but just thought I should mention: looking at this, I think it would be better to remove the `| safe` entirely, unless we know for a fact that there is some exception where we want to allow raw HTML through as cells in AtomicTableBlocks. I doubt we do. I took a quick look at doing an audit of the existing pages with ATBs to see if we do that anywhere, but it's unfortunately non-trivial to do that right now.

Reply:
The problem that re-ordering these filters fixes is that in Python 3, the table cells themselves were coming through as escaped HTML rather than raw HTML (I think because `linesbreakbr` was after `safe` and Jinja2 seems to do some different safe-string handling in Python 3 than Python 2). That suggests that there's a lot more refactoring we probably *should* do to our table blocks, but that seems like a mini-project in itself from my dive into why it was happening.